hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -12,6 +12,11 @@ Prevent Popups to overflow tables.
     if ($button.hasClass('btn-slide-active')) {
       $popup = $button.children('ul.panel');
 
+      // Use id of parent tr tag
+      $popup.attr("id", $button.closest('.crm-entity')[0].id);
+      // Use class of parent tr tag
+      $popup.addClass($($button.closest('.crm-entity')[0]).attr("class"));
```

Comment:
Why do you need to add the class as well? I thought the crm script only looked for the `id`?